README.md

```diff
@@ -233,6 +233,15 @@ If you used Windows and decided to switch to Linux, this course should help you
 * [Bash Guide for Beginners](https://linux.die.net/Bash-Beginners-Guide/)
 * [Advanced Bash-Scripting Guide](https://linux.die.net/abs-guide/)
 
+### ZSH Useful Resources
+* [antigen](https://github.com/zsh-users/antigen)
+* [oh-my-zsh](http://ohmyz.sh/)
+* [Outrageously Useful Tips To Master Your Z Shell](http://reasoniamhere.com/2014/01/11/outrageously-useful-tips-to-master-your-z-shell/)
+* [awesome-zsh-plugins-list](https://github.com/unixorn/awesome-zsh-plugins)
+* [Why ZSH is Cooler than your Shell](http://www.slideshare.net/jaguardesignstudio/why-zsh-is-cooler-than-your-shell-16194692)
+* [The Text Triumvirate](http://www.drbunsen.org/the-text-triumvirate/) - Seth Brown's tutorial on combining zsh, tmux and vim.
+The [ZSH Reference Card](http://www.bash2zsh.com/zsh_refcard/refcard.pdf) and [zsh-lovers site](http://grml.org/zsh/zsh-lovers.html) are indispensable.
```

Comment:
Separate this into two lines and indicate that the reference card is a PDF somehow so that people wouldn't be surprised when they click it. Also, continue the list with `*` at the beginning.